cmd/cmd_test.go

```diff
@@ -521,7 +521,7 @@ func TestOptionsSternConfig(t *testing.T) {
 				c.AllNamespaces = true
 				c.LabelSelector = labelSelector
 				c.FieldSelector = fieldSelector
-				c.TailLines = pointer.Int64(10)
+				c.TailLines = ptr.To[int64](10)
```

Comment:
:memo: The `pointer` package was deprecated in https://github.com/kubernetes/utils/commit/dc1a5d8ab3834835ab2294895323df879d7192ba.